integration/image_create_test.go

```diff
@@ -135,9 +125,7 @@ const imageCreateResponse = `{
 	  "error_message": "",
 	  "id": 38413969,
 	  "name": "ubuntu-18.04-minimal",
-	  "regions": [
-  
-	  ],
+	  "regions": [],
```

Comment:
nit: should this actually match the same region we requested ie `nyc3`. It's just fake output so probably fine?

Reply:
Hmm I think I'll add the region. Better to have as close to real data as possible in these so it'll make our lives easier once we start on end-to-end tests.

Reply:
That's what the response to a create request actually looks like. It seems that the `region` attribute remains empty until the `status` changes to `available`.

Reply:
On second though, the API docs seem to suggest a region isn't returned here... time to test it out

Reply:
Regions is empty:

```json
{
  "image":{
    "created_at":"2019-10-04T17:03:09Z",
    "description":"Cloud-optimized image w/ small footprint",
    "distribution":"Ubuntu",
    "error_message":"",
    "id":53120417,
    "name":"ubuntu-18.04-minimal-test-2",
    "regions":[

    ],
    "status":"NEW",
    "tags":[

    ],
    "type":"custom",
    "updated_at":"2019-10-04T17:03:09Z"
  }
}
```

Reply:
Whoops, missed asb's comment, guess I didn't need to test 😛